debs/generate_debian_package.sh

```diff
@@ -74,7 +75,7 @@ build_deb() {
         ${CUSTOM_CODE_VOL} \
         ${CONTAINER_NAME} ${TARGET} ${BRANCH} ${ARCHITECTURE} \
         ${REVISION} ${JOBS} ${INSTALLATION_PATH} ${DEBUG} \
-        ${CHECKSUM} ${PACKAGES_BRANCH} ${USE_LOCAL_SPECS} ${LOCAL_SOURCE_CODE} || return 1
+        ${CHECKSUM} ${PACKAGES_BRANCH} ${USE_LOCAL_SPECS} "${LOCAL_SOURCE_CODE}" "DATABASE=${DATABASE_OUTPUT}" || return 1
```

Comment:
```shell
...
DATABASE_FLAG="DATABASE=${DATABASE_OUTPUT}"
...

        ${CHECKSUM} ${PACKAGES_BRANCH} ${USE_LOCAL_SPECS} \ 
        ${LOCAL_SOURCE_CODE} ${DATABASE_FLAG} || return 1
```
To keep the consistency with other configuration variables, you need to set a variable for the `DATABASE` flag and then use it in the `docker run` line. Also, break the line to make easier reading the command.